open-metadata-test/open-metadata-fvt/access-services-fvt/subject-area-fvt/src/test/java/org/odpi/openmetadata/accessservices/subjectarea/fvt/junit/AllIT.java

```diff
@@ -0,0 +1,61 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+
+package org.odpi.openmetadata.accessservices.subjectarea.fvt.junit;
+
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
+import org.odpi.openmetadata.accessservices.subjectarea.fvt.*;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+@Execution(ExecutionMode.SAME_THREAD)
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
```

Comment:
and this annotation. I misread as being default. :-)

Reply:
I initially thought so too. It is strange that this is not so)